src/components/slider/slider.scss

```diff
@@ -0,0 +1,125 @@
+@import 'default-theme';
+
+$slider-size: 20px !default;
+$slider-min-size: 128px !default;
+
+$slider-track-height: 2px !default;
+$slider-thumb-width: 20px !default;
+$slider-thumb-height: $slider-thumb-width !default;
+
+$slider-thumb-default-scale: 0.7 !default;
+$slider-thumb-focus-scale: 1 !default;
+
+// TODO(iveysaur): Find an implementation to hide the track under a disabled thumb.
+$slider-off-color: rgba(black, 0.26);
+$slider-focused-color: rgba(black, 0.38);
+$slider-disabled-color: rgba(black, 0.26);
+
+@function center-vertically($outerHeight, $innerHeight) {
+  @return ($outerHeight / 2) - ($innerHeight / 2);
+}
+
+@mixin slider-thumb-position($width: $slider-thumb-width, $height: $slider-thumb-height) {
+  position: absolute;
+  top: center-vertically($slider-size, $height);
+  width: $width;
+  height: $height;
+  border-radius: max($width, $height);
+}
+
+md-slider {
+  height: $slider-size;
+  min-width: $slider-min-size;
+  position: relative;
+  padding: 0;
+  display: inline-block;
+  outline: none;
+  vertical-align: middle;
+}
+
+md-slider *,
+md-slider *::after {
+  box-sizing: border-box;
+}
+
+.md-slider-wrapper {
```

Comment:
Add comments for `slider-wrapper` and `slider-container` since it's not immediately obvious what they're used for.